app/presenters/summary/html_sections/without_notice_details.rb

```diff
@@ -0,0 +1,30 @@
+module Summary
+  module HtmlSections
+    class WithoutNoticeDetails < Sections::BaseSectionPresenter
+      def name
+        :without_notice_hearing
+      end
+
+      def answers
+        [
+          Answer.new(:without_notice_hearing, c100.without_notice, change_path: edit_steps_application_without_notice_path),
+          AnswersGroup.new(:without_notice_hearing_details, without_notice_hearing_details, change_path: edit_steps_application_without_notice_details_path),
+        ].select(&:show?)
+      end
+
+      private
+
+      def without_notice_hearing_details
+        return [] unless c100.without_notice.eql?(GenericYesNo::YES.to_s)
```

Comment:
No need for this, the `AnswersGroup` will only render what is available, otherwise, will render nothing.